crates/sui-sdk/src/crypto.rs

```diff
@@ -64,9 +63,9 @@ impl Display for KeystoreType {
     }
 }
 
-#[derive(Serialize, Deserialize, Default)]
```

Comment:
i dont see an usage of this, safe to remove?

Reply:
looks safe to me

Reply:
Ditto